ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthPeerTest.java

```diff
@@ -402,18 +421,30 @@ private void messageStream(
   }
 
   private EthPeer createPeer() {
-    return createPeer(Collections.emptyList());
+    return createPeer(Collections.emptyList(), Collections.emptyList());
+  }
+
+  private EthPeer createPeerWithPermissioningProvider(
+      final List<NodeMessagePermissioningProvider> permissioningProviders) {
+    return createPeer(Collections.emptyList(), permissioningProviders);
   }
 
   private EthPeer createPeer(final PeerValidator... peerValidators) {
-    return createPeer(Arrays.asList(peerValidators));
+    return createPeer(Arrays.asList(peerValidators), Collections.emptyList());
   }
 
-  private EthPeer createPeer(final List<PeerValidator> peerValidators) {
-    final Set<Capability> caps = new HashSet<>(singletonList(EthProtocol.ETH63));
-    final PeerConnection peerConnection = new MockPeerConnection(caps);
```

Comment:
caps not needed?

Reply:
Surprisingly not! It doesn't do anything in this test. Changed it to use `mock(PeerConnection.class)` nothing is being validated against, it's just there to make you don't get null pointers.